assembl/static2/js/app/integration/brightMirror/index.jsx

```diff
@@ -1,18 +1,48 @@
 import React, { Component } from 'react';
 
 import Aux from '../../hoc/aux/aux';
+import { customFictionsList } from '../../stories/components/debate/brightMirror/fictionsList.stories';
+import { customInstructionsText } from '../../stories/components/debate/brightMirror/instructionsText.stories';
+import FictionsList from '../../components/debate/brightMirror/fictionsList';
+import InstructionsText from '../../components/debate/brightMirror/instructionsText';
+import TopPostFormContainer from '../../components/debate/common/topPostFormContainer';
+
+const timeline = [
+  {
+    identifier: 'brightMirror',
+    start: '1900-01-01T02:00:00Z',
+    end: '2200-01-01T00:00:00Z',
+    title: { entries: [{ en: 'brightMirror' }] }
```

Comment:
Actually a phase retrieved by graphql doesn't have this format anymore, title is just a string. `title: 'brightMirror'`